### DISCUSSION TITLE
(NPM) Vulnerabilities are added daily since Friday

### DISCUSSION THREAD

**Discussion by ichistmeinname** [DependencyTrack/dependency-track#1452] [Q&A] [answered]
Version: Dependency-Track v4.3.6

We're running the software for a good month now and starting Friday, the vulnerabilities coming from the "NPM Audit" are readded daily (with different URLs, with only the latest referencing the GitHub advisory correctly, all early links are dead).
Is this a problem with NPM advisories merging with GitHub? But then again, why did it start on Friday?
We're still watching if the behaviour changes, but it's now the third day (every!) npm finding gets reposted. The following screenshot shows only one vulnerability that was added four times.

![Screen Shot 2022-03-08 at 09 54 33](https://user-images.githubusercontent.com/1886946/157201972-0a853a63-5a65-4179-81e2-3c1e233fc2de.png)

**Comment by stevespringett** [comment, marked as answer]
Likely an issue with GitHub. Also note that NPM Mirroring was suppose to have stopped working as GitHub stated they were shutting it down. Not sure if that's happened yet or not. All DT users are strongly encouraged to upgrade to 4.4.x which removes support for NPM Advisories in favor of GitHub Advisories. Unfortunately, GitHub did not perform any mappings between their advisories and NPM advisories.

**Reply by ichistmeinname** [reply]
Thanks for the quick reply! We're already discussing just updating to v4.4, but I was wondering why nobody else mentioned this "hiccup", so I thought I'd just ask ; )

**Reply by ichistmeinname** [reply]
Thanks again; the update resolved the issue!
